exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function CreateGreeting(name) {
+  return `HELLO ${name}`;
+}
+
+function ToUpper(string) {
+  return CreateGreeting(string.toUpperCase());
+}
+
+console.log(ToUpper(mentor1));
+console.log(ToUpper(mentor2));
+console.log(ToUpper(mentor3));
+console.log(ToUpper(mentor4));
+console.log(ToUpper(mentor5));
+
+console.log("\n");
```

Comment:
really neat work Georgina, sorry the last line of the coding, in console.log you put new line sign, can you please explain how it worked :),

Reply:
Thanks Uzma, I'm learning something new each day. Using "/n" in the console.log lists each on a new line. I also learnt that you can even add some sort of margin at the start of each line if you list items like this  - example: 
console.log("one", "\n","two");